src/palace/manager/api/circulation.py

```diff
@@ -570,6 +570,33 @@ class BaseCirculationApiSettings(BaseSettings):
         )
     }
 
+    subscription_activation_date: str | None = FormField(
```

Comment:
Since these are coming in as dates, should we be setting the type here to reflect that, rather then just a generic `str`?

Reply:
I did try using `subscription_activation_date: datetime.date | None = FormField( ...` for these, but ran into type errors in Pydantic.

I can dig into that some more as I do the implementation.